docs/cuda-miner.md

```diff
@@ -82,17 +82,15 @@ sudo apt-get update && sudo apt-get install cuda
 
 ### Miner install
 
-At this point the CUDA toolkit is installed. Next step is to build the cuckoo CUDA miner. If the node has been installed (build) from source, the same source tree can be used. Otherwise if the binary package has been used for installation, **the same version** of epoch source code must be downloaded.
+At this point the CUDA toolkit is installed. Next step is to build the cuckoo CUDA miner. If the node has been installed (build) from source, the same source tree can be used. Otherwise if the binary package has been used for installation, **the same version** of Aeternity node's source code must be downloaded.
 
-Epoch source code can be downloaded by cloning the git repository:
+Aeternity node's source code can be downloaded by cloning the git repository:
 ```bash
 cd ~
-git clone https://github.com/aeternity/epoch.git epoch && cd epoch
+git clone https://github.com/aeternity/aeternity.git aeternity_source && cd aeternity_source ## Before version 1.3.0, the repository used to be https://github.com/aeternity/epoch.git
```

Comment:
I don't think it needs the clarification as it's backward compatible i.e. you can checkout old tags.

Reply:
My problem was that this section claims to have been tested:
> The documentation in this section is tested on:
> - Epoch version 1.0.0-rc2

Hence I am trying not to misrepresent that.